MovieTime/MovieTime/Model/MTViewModel.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  MTViewModel.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 09/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+class MTViewModel: NSObject {
```

Comment:
in a future version of this app, what is this view model intended for?

Reply:
I see two possibilities for this.
1 - The viewmodel should have the same characteristics, as they are "linked" to a "view". And for this, it is possible to have implementations for all viewModels, like an common initializer, a property or something like that.
2 - But it is possible to do all this with a protocol, `MNViewModelProtocol`, since protocols define the behavior of the object, and that is what is needed here.
So, I could make that change in the project.

Reply:
ideally this class would not exist until it becomes necessary ;)